README.md

```diff
@@ -5,3 +5,4 @@
 ### Fixed a new issue 
 
 ### This documentation is fixed 
+## A new file system has been added to the git 
```

Comment:
This line should be removed in future release

Reply:
this file will be removed in next release

Reply:
this code has been changed based on this pull request 
#8